CONTRIBUTING.md

```diff
@@ -101,9 +101,9 @@ The following is the list of supported scopes:
   - `web-react`
   - `web-twig`
 - Repository-wide:
-  - `ci`: used for changes that affects Continuous Integration process and builds
-  - `repo`: used for repository wide changes
-  - none/empty string: useful for `Test` and `Refactor` changes that are done across all packages (e.g. `Test: Add missing unit tests`) and for docs changes that are not related to a specific package (e.g. `Docs: Fix typo in tutorial`).
+  - `ci`: used for changes that affect the Continuous Integration process and builds
+  - `repo`: used for repository-wide changes
+  - none/empty string: useful for `Test` and `Refactor` changes that are done across all packages (e.g. `Test: Add missing unit tests`) and for docs changes that are not related to a specific package (e.g. `Docs: Fix `typo in `the `tutorial`).
```

Comment:
backtick madness. Please fix.

Reply:
Hmm, Grammarly extension fixes. Always broke something. :-/

Reply:
Thanks for noticing.